homeassistant/components/google_photos/services.yaml

```diff
@@ -0,0 +1,11 @@
+upload:
+  fields:
+    config_entry_id:
+      required: true
+      selector:
+        config_entry:
+          integration: google_photos
+    filename:
+      required: false
+      selector:
+        text:
```

Comment:
Wasn't this a list?

Reply:
Yes, good catch. Updated to use `object:` which i think is right..